index.html

```diff
@@ -38,10 +38,20 @@ <h1 class="logo"><em>HotMaps</em></h1>
     </div>
 
 <!-- Google Maps here -->
-    <div class="col s12 m8 l9"> <!-- Note that "m8 l9" was added -->
-      <div id="map">
-
-      </div>
+    <div class="col s12 m8 l9">
+
+      <!-- Adding the search bar -->
+      <nav class="col xl6 l6 m6 s9 offset-xl1 offset-l1 offset-m1 offset-s1 searchNav hoverable teal darken-1">
+        <div class="nav-wrapper  teal darken-1">
+          <form>
+            <div class="input-field">
+              <input class=" teal darken-1" id="search" type="text" required>
+              <!-- <label class="label-icon" for="search"><i class="material-icons">search</i></label> -->
+            </div>
+          </form>
+        </div>
+      </nav>
+        <div class="col s12" id="map"></div>
     </div>
 
     <!-- Tap Target Structure -->
```

Comment:
Thought you were missing a closing </div>, but I found it hiding below!

Looks good.